# Get a servo encoder filter's current position count. 
int32 servo_id
---
int32 count
